core/src/Cabana_AoSoA.hpp

```diff
@@ -310,6 +310,9 @@ class AoSoA
     */
     void resize( const size_type n )
     {
+#if !defined( __HIP_DEVICE_COMPILE__ )
+        using std::floor;
```

Comment:
`hipcc` doesn't have `floor` in host-only functions?

Reply:
Good catch.  Just had grep'ed for `floor` and changed everywhere w/o thinking.  Will revert.